### PR TITLE
chore(main): switch over to main

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

**Review by mtomcal — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        node-version: ${{ matrix.node }}
+    - name: Install Dependencies
+      run: npm ci
+      env:
+        NODE_ENV: development
+    - name: Unit Tests
+      run: npm run test:unit
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:32]: Do we need this token at every stage? Or in this workflow at all?

```suggestion
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Git History Test
+      run: npm run test:git-history
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:37]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Lockfile Lint Test
+      run: npm run test:lockfile
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:42]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Lint
+      run: npm run test:lint
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:47]:
```suggestion
```

**Review by Francois-Esquire — CHANGES_REQUESTED** [review]
Would be safer IMO to avoid exposing the token if we don't need to...

File: .github/workflows/health-check.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        node-version: ${{ matrix.node }}
+    - name: Install Dependencies
+      run: npm ci
+      env:
+        NODE_ENV: development
+    - name: Run Test Script
+      run: npm run test
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/health-check.yml:32]:
```suggestion
```

**Review by Francois-Esquire — APPROVED** [review]

**Pull request merged**